app/Http/Controllers/Auth/RegisteredUserController.php

```diff
@@ -47,6 +48,7 @@ public function store(Request $request)
             'mobile_number' => $request->mobile_number,
             'mobile_verify_code' => random_int(111111, 999999),
             'mobile_attempts_left' => config('mobile.max_attempts'),
+            'mobile_last_send' => Carbon::now(),
```

Comment:
@MaxRomagnoli this could be only `now()` helper, without importing `Carbon\Carbon`

Reply:
I think Carbon::now() could be better because it uses the utc configuration put in the config variables, isn't?

Reply:
@MaxRomagnoli no, it's absolutely the same behavior, just shorter.